gnd/src/main/java/com/google/android/gnd/ui/util/FileUtil.java

```diff
@@ -70,6 +70,13 @@ public File getFile(String filename) throws FileNotFoundException {
     return file;
   }
 
+  /**
+   * Creates a new empty file in the app's file directory /data/data/com.google.android.gnd/files
+   */
+  public File makeFile(String filename) {
```

Comment:
makeFile -> createFile?

In the comment, you've mentioned that it creates a new empty file. But it is not always going to be true if the file for the provided filename already exists. Please consider adding a check to validate that no file exists for the given name or update comment.

Reply:
I've renamed it to `getOrCreateFile` and updated the comment accordingly, thanks! e17a280